synapse/cores/common.py

```diff
@@ -524,7 +527,8 @@ def _cortex_condefs():
             ('log:save', {'type': 'bool', 'asloc': 'logsave', 'defval': 0,
                           'doc': 'Enables saving exceptions to the cortex as syn:log nodes'}),
             ('log:level', {'type': 'int', 'asloc': 'loglevel', 'defval': 0, 'doc': 'Filters log events to >= level'}),
-            ('modules', {'defval': (), 'doc': 'An optional list of (pypath,conf) tuples for synapse modules to load'})
+            ('modules', {'defval': (), 'doc': 'An optional list of (pypath,conf) tuples for synapse modules to load'}),
+            ('membranes', {'defval': (), 'doc': 'An optional list of (name,rules) tuples to create Membranes with'})
```

Comment:
@invisig0th - what were you thinking for how to specify what to do with the events? Right now, membranes just take the events that pass and put them into a named fifo in the core. Even if the behavior is that the events are ALWAYS spliced into another core, we need some way to specify the target core.

Reply:
We decided that this PR should only filter events and put them into a fifo.